### DISCUSSION TITLE
v0.12.0-beta.5

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#761] [Releases]
## What's Changed
* [Chore] Hint links by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/759
* Release v0.12.0-beta.5 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/760

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.12.0-beta.4...v0.12.0-beta.5

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.12.0-beta.5'>v0.12.0-beta.5</a>.</em>

**Comment by jiriteach** [comment]
Thanks.

Out of interest - how long do the builds take to complete? Saw the release pop-up but still waiting for the image. 
https://github.com/alexjustesen/speedtest-tracker/pkgs/container/speedtest-tracker

**Reply by alexjustesen** [reply]
Like 10min which is painful, it's because it has to build images for 4 different architecture types.

**Comment by jiriteach** [comment]
Working well - BTW am still seeing encryption key errors on first startup. Have to restart the container for things to work. This is on RPI's.

**Reply by alexjustesen** [reply]
It should now just throw a ⚠️ warning and not an error and still start correctly. Let me know if that doesn't happen.

**Reply by jiriteach** [reply]
Its throwing errors - did a fresh install before and threw the same error. Restart of container worked.

**Reply by alexjustesen** [reply]
That restart is killing me, thinking it's something about config caching

**Reply by ShlomiD83** [reply]
@Jiriteach you have an APP_KEY env variable set?

**Reply by jiriteach** [reply]
Yup I do in the .env file

**Reply by ShlomiD83** [reply]
@Jiriteach try adding it to your stack/compose and delete your env file.
that's what worked for me.

**Reply by jiriteach** [reply]
Could try that. It's not a biggie for me - it's just happens the first time a new version is updated. Container is launched then needs to be restarted before everything is working fine.

**Reply by ShlomiD83** [reply]
that's exactly problem I was having, setting the APP_KEY as a stack env variable and removing the .env file solved my issue.

**Comment by jiriteach** [comment]
Just on the performance side especially on one of my RPI 4's. Noticed one of mine was substantially faster than the other. They are both RPI 4's. The one that is substantially faster was running RPI OS 64-bit while the other was running Ubuntu 64-bit. Reinstalled the Ubuntu one to run RPI OS 64-bit and Speedtest Tracker is substantially faster so must be something Laravel and Filament.

**Reply by alexjustesen** [reply]
I've seen this too and with other containers. I think it's because rpi OS is far simpler than Ubuntu and is tuned specifically for that hardware. Don't think it's Laravel/Filament related but I'll keep an eye out
